Local patch to add pointers to our SMF services into upstream man pages.

Not appropriate for upstream, since it's specific to our SMF & IPS setup.

--- a/docs/reference/gio/glib-compile-schemas.rst	2025-08-22 05:29:34.000000000 -0700
+++ b/docs/reference/gio/glib-compile-schemas.rst	2025-10-02 11:42:29.351804836 -0700
@@ -50,6 +50,14 @@
 override is made in a file numbered 10 and then again in a file numbered 20, the
 override from 20 will take precedence).
 
+In Oracle Solaris, the transient ``glib-compile-schemas`` service runs when
+necessary to update the ``gschemas.compiled`` file with information about the
+GSettings XML schema and vendor override files.  
+
+See the ``application/desktop-cache/glib-compile-schemas`` man page for
+details about the ``glib-compile-schemas`` service.  See the ``smf``
+man page for information about the Oracle Solaris Service Management Facility.
+
 OPTIONS
 -------

--- a/docs/reference/gio/gio-querymodules.rst	2025-10-08 13:47:07.063214745 -0700
+++ b/docs/reference/gio/gio-querymodules.rst	2025-10-08 13:51:53.581262070 -0700
@@ -28,3 +28,17 @@
 that has been found. It is used by GIO at runtime to avoid opening all modules
 just to find out which extension points they are implementing.
 
+In Oracle Solaris, the transient ``gio-module-cache`` service runs 
+``gio-querymodules`` when necessary to update the ``giomodule.cache`` file
+with information about the installed GIO extension modules.  And example of 
+when the ``gio-module-cache`` service runs is when a package that specifies
+the ``svc:/application/desktop-cache/gio-module-cache`` service as the value
+of a ``restart_fmri`` attribute on is being installed.  A package that installs
+a GIO extension module should have such an attribute.
+ 
+See the ``application/desktop-cache/gio-module-cache`` man page for details
+about the ``gio-module-cache`` service.  See the ``smf`` man page for
+information about the Oracle Solaris Service Management Facility.
+
+GIO modules are usually installed in the ``gio/modules`` subdirectory of libdir.
+
